### DISCUSSION TITLE
Close a tab and return to previous tab but not next ?

### DISCUSSION THREAD

**Discussion by franzhuang** [qutebrowser/qutebrowser#6747] [General] [answered]
For example: I open 3 tabs.

Tab_1,  Tab_2,  Tab_3

I now in Tab_1 and open and go to a new tab_A:

Tab_1,  Tab_A,  Tab_2,  Tab_3

I close Tab_A after browsering, and it bring me to Tab_2.
But I want to go back to previous tab--Tab_1, rather than next tab.
If there is configuration option to do this?

**Comment by toofar** [comment, marked as answer]
Try `:set tabs.select_on_remove prev`, assuming you are loading you autoconfig.yml (the default).

That setting is a bit hard to find because we use "position" and "close" more than "select" and "remove" in settings.

**Reply by franzhuang** [reply]
Thank you,it's the one config I need.

**Reply by The-Compiler** [reply]
Probably would be good to rename that setting, I agree it's a bit inconsistent. I opened https://github.com/qutebrowser/qutebrowser/issues/6754.
